polaris-react/src/components/BulkActions/BulkActions.tsx

```diff
@@ -285,9 +325,12 @@ class BulkActionsInner extends PureComponent<CombinedProps, State> {
 
     const groupContent =
       promotedActionsMarkup || actionsPopover ? (
-        <InlineStack gap="300">
-          {promotedActionsMarkup}
-          {actionsPopover}
+        <InlineStack gap="300" blockAlign="center">
+          <CheckableButton {...checkableButtonProps} />
+          <InlineStack gap="100" blockAlign="center">
```

Comment:
```suggestion
          <InlineStack gap="200" blockAlign="center">
```

Nit: we tend to have 8px of spacing when buttons are rendered next to each other (i.e., [ButtonGroup](https://storybook.polaris.shopify.com/?path=/story/all-components-buttongroup--default)) so light suggestion to have this be consistent by updating the gap.

Reply:
@laurkim I wouldn't want to buck the trend normally, but we have a `100` gap between the buttons and the edge of the bulk actions so having the gap between buttons as `100` helps keep balance. I've attached a screenshot of how it looks at `200` and it just feels a teeny bit off
<img width="539" alt="Screenshot 2024-01-05 at 13 24 22" src="https://github.com/Shopify/polaris/assets/2562596/e2d58e72-e3d1-4992-89c0-428dc561cfe3">

Reply:
@mrcthms makes sense to me, sounds good to just leave it at 100 then! 🤝